README.md

```diff
@@ -1,3 +1,18 @@
+# Important Notice
+
+The contents of this repository have been consolidated into [tinymist](https://github.com/Myriad-Dreamin/tinymist). It is an all-in-one language server for typst.
+
+
+We recommend all users migrate to tinymist for the following benefits:
```

Comment:
Do we need to mention the instructions to migrate?
- We still maintain the typst-preview extension for a while (until we allow run lsp with only typst-preview feature)
   - The lazy people can continue using their setting, as all old things are still working.
   - This respect people who love minimal env, like a treesitter plugin plus preview.
- We ensure compatibility to typst-preview as much as possible.
   - for vscode users: uninstall the preview extension and install the tinymist extension.
   - for standalone program users: `typst-preview -> tinymist preview`